Improve testplan minor release

gabi18 opened the pull request:
Added some missing 'heavy_check_mark' and 'construction'

gabi18 commented:
@Talank Could you please check esp. 6 / 2 (does the x mean, the test failed/still fails?) and 9.1 / 5 (test also fails?) and confirm that all automated tests now are marked correctly.

swoichha commented:
@gabi18 I'll look into it and get back to you.

swoichha commented:
@gabi18 we did not add the :x: in 6/2. I think it was there while we manually tested for `2.9`. For 9.1/5 we just added a :robot: and the name of the file where the test have been added (i.e. `tst_sharing`). Both of these tests have been automated and it passes on CI as well.

All the gui tests that fails are either skipped or are reported in CI logs. Also, we have not completely edited the file which marks the tests that are automated. Hopefully after we close this issue https://github.com/owncloud/client/issues/8971 we can confirm which tests have automated.

gabi18 commented:
> @gabi18 we did not add the x in 6/2. I think it was there while we manually tested for `2.9`. For 9.1/5 we just added a robot and the name of the file where the test have been added (i.e. `tst_sharing`). Both of these tests have been automated and it passes on CI as well.
Thanks for having a look. I have added the heavy_check_mark to  6/2.  and 9.1/5.
> 
> All the gui tests that fails are either skipped or are reported in CI logs. Also, we have not completely edited the file which marks the tests that are automated. Hopefully after we close this issue [owncloud/client#8971](https://github.com/owncloud/client/issues/8971) we can confirm which tests have automated. cc: @Talank
@swoichha  @Talank 
If you agree I will merge this pull request now to have an overview which tests are already automated. More tests can be added later.

swoichha reviewed (approved):
@gabi18 This looks good to me we can merge this. But, I think we need to make an extra column for squish test where we can add :heavy_check_mark: :x: :construction: so that we can have two column one for squish tests and one for manual tests.

Talank reviewed (approved):
like @swoichha said, we can discuss that later on how to distinguish tests that are automated. For now, changes made in this PR looks good to me as well.

gabi18 commented:
@swoichha @Talank Thanks for your review and ideas. My idea concerning the Result column was to use it for automated and manual tests. The result for a squish test is always :heavy_check_mark:, all failures must be fixed before building the beta/RC candidates. We have added to the prerequisites of the release ticket template that all squish tests have to be successful https://github.com/owncloud/client/blob/master/.github/patch_release_template.md.
We will see how the test plan works for the next release and can apply improvements.
